hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -207,7 +220,7 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
     this.updatePieChart('/', DEFAULT_DISPLAY_LIMIT);
   }
 
-  clickPieSection(e, curPath: string): void {
+  clickPieSection(e, curPath: string): void { 
```

Comment:
nit, please remove the unnecessary space here

Reply:
Changed in Next Commit.